lib/saluki-app/src/logging.rs

```diff
@@ -12,19 +12,45 @@ pub fn fatal_and_exit(message: String) {
 /// Initializes the logging subsystem for `tracing`.
 ///
 /// This function reads the `DD_LOG_LEVEL` environment variable to determine the log level to use. If the environment
-/// variable is not set, the default log level is `INFO`.
+/// variable is not set, the default log level is `INFO`. Additionally, it reads the `DD_LOG_FORMAT` environment
+/// variable to determine which output format to use. If it is set to `json` (case insensitive), the logs will be
+/// formatted as JSON. If it is set to any other value, or not set at all, the logs will default to a rich, colored,
+/// human-readable format.
 ///
 /// ## Errors
 ///
 /// If the logging subsystem was already initialized, an error will be returned.
 pub fn initialize_logging() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
+    let is_json = std::env::var("DD_LOG_FORMAT")
```

Comment:
Oh interesting, TIL!

Yeah, I'll make it match. I still plan on sort of setting it `true` by default for internal deployments because otherwise the `tracing` format is a little too wonky to easily parse on the DD side... but it should definitely match in terms of environment variable name, so good catch.